fidesctl/src/fidesctl/core/api_helpers.py

```diff
@@ -67,3 +67,29 @@ def get_server_resource(
     )
 
     return server_resource
+
+
+def list_server_resource(
+    url: str,
+    resource_type: str,
+    headers: Dict[str, str],
+) -> List[FidesModel]:
```

Comment:
It would improve the usage if this function always returned a list. Maybe `[]` instead of `None`? Otherwise, `None` should be specified as a possible return type here.

Reply:
updated the docstring here and noted the behavior